modules/local/compute_gene_scores/environment.yml

```diff
@@ -0,0 +1,16 @@
+name: compute_gene_scores
```

Comment:
name is also removed in the new nf-core `environment.yml` specification convention.